### PR TITLE
Add regressions for uniCOIL noexp on MS MARCO v1 corpora

### PR DESCRIPTION
**Pull request by lintool**
hey @MXueguang can you do a sanity check against results on your end?

### CODE REVIEW

File: docs/regressions-msmarco-passage-unicoil-noexp.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+With the above commands, you should be able to reproduce the following results:
+
+| MAP                                                                                                          | uniCOIL w/ doc2query-T5 expansion|
+|:-------------------------------------------------------------------------------------------------------------|-----------|
+| [MS MARCO Passage: Dev](https://github.com/microsoft/MSMARCO-Passage-Ranking)                                | 0.3215    |
+
+
+| MRR                                                                                                          | uniCOIL w/ doc2query-T5 expansion|
+|:-------------------------------------------------------------------------------------------------------------|-----------|
+| [MS MARCO Passage: Dev](https://github.com/microsoft/MSMARCO-Passage-Ranking)                                | 0.3263    |
```
Comment (MXueguang) [inline docs/regressions-msmarco-passage-unicoil-noexp.md:99]: MRR for msmarco-passage should use -M 100 ?

File: docs/regressions-dl19-passage-unicoil-noexp.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+tools/eval/trec_eval.9.0.4/trec_eval -m ndcg_cut.10 -c src/main/resources/topics-and-qrels/qrels.dl19-passage.txt runs/run.msmarco-passage-unicoil-noexp.unicoil.topics.dl19-passage.unicoil-noexp.0shot.txt
+tools/eval/trec_eval.9.0.4/trec_eval -m recall.100 -c -l 2 src/main/resources/topics-and-qrels/qrels.dl19-passage.txt runs/run.msmarco-passage-unicoil-noexp.unicoil.topics.dl19-passage.unicoil-noexp.0shot.txt
+tools/eval/trec_eval.9.0.4/trec_eval -m recall.1000 -c -l 2 src/main/resources/topics-and-qrels/qrels.dl19-passage.txt runs/run.msmarco-passage-unicoil-noexp.unicoil.topics.dl19-passage.unicoil-noexp.0shot.txt
+```
+
+## Effectiveness
+
+With the above commands, you should be able to reproduce the following results:
+
+| AP@1000                                                                                                      | uniCOIL w/ doc2query-T5 expansion|
```
Comment (MXueguang) [inline docs/regressions-dl19-passage-unicoil-noexp.md:99]: uniCOIL w/ doc2query-T5 expansion, should be uniCOIL noexp

**Comment by MXueguang** [comment]
there are slightly difference but within 0.001

File: docs/regressions-msmarco-passage-unicoil-noexp.md (hunk 1 above)
Comment (lintool) [reply docs/regressions-msmarco-passage-unicoil-noexp.md:99]: This is okay... this isn't the same as the MRR metric computed by the official MS MARCO script.  Should we just remove to reduce confusion?

MRR@k computed by `trec_eval` is not the same as the score computed by the official scripts...

**Review by MXueguang — APPROVED** [review]

**Pull request merged**